src/main/resources/static/js/subscription.js

```diff
@@ -459,11 +460,13 @@ jQuery(document).ready(function () {
                     "title": "Action",
                     "data": null,
                     "render": function (data, type, row, meta) {
-
-                        if (isSecured == false || (row.ldapUserName == currentUser && row.ldapUserName != null)) {
+                    	var unsecureEditAll = isSecured == false;
+                    	var securedEditOwn = (row.ldapUserName == currentUser && row.ldapUserName != null);
+                    	var securedEditNoOwner = isSecured == true && row.ldapUserName.length == 0 && currentUser != null;
```

Comment:
var securedEditNoOwner = isSecured == true && row.ldapUserName.length == 0 && currentUser != null;
This variable may be joined with "unsecuredEditAll", the variable may be named "allMayEdit" and check only if "wor.ldapUserName.length == 0", this means that no matter if secured or not anyone may edit a row without userName

Reply:
okk